.gitignore

```diff
@@ -113,4 +113,4 @@ database
 src/modules/data/types
 
 # Database files
-data/
+/data/
```

Comment:
Isn't it a folder?

Reply:
It means everything under this path. `data/` match any data folder in the repo (including `tests/data/`) while `/data/` only match `data/` at the root of the repository.

Reply:
Yes,
But in my case the root data folder at root `/data/` was not ignored by git.

Reply:
Could you demonstrate how it doesn't work for you ?

Reply:
https://github.com/bpfkorea/stoa/pull/156
There used to be `stoa/logs/` in front of.
I think it was a different case.